ros2/source/source/Dockerfile

```diff
@@ -52,16 +55,18 @@ RUN apt-get update && apt-get install -q -y \
     python3-pip \
     python3-pyparsing \
     python3-setuptools \
-    python3-vcstool \
+    # python3-vcstool \
     python3-yaml \
     qtbase5-dev \
     uncrustify \
     wget \
+    xorg-dev \
```

Comment:
What needs `xorg-dev`? Is this for `rviz_ogre_vendor`? It built fine for me after only installing `libxrandr-dev`.

Reply:
Thanks for checking that. Yeah narrowing down what's needed to build and run rviz deps and tests is the last remaining item for the bionic migration. Once confirmed on the ros2/ci PR we can push the corresponding changes here